src/tz390.java

```diff
@@ -3348,11 +3361,85 @@ public void init_tz390(){
+                                                                                                         // #503
+    machine_option_id = new String[machine_optable_equivalence.length];                                  // #503
+    machines_optable  = new String[machine_optable_equivalence.length];                                  // #503
+    optable_option_id = new String[optable_optable_equivalence.length];                                  // #503
+    optables_optable  = new String[optable_optable_equivalence.length];                                  // #503
+                                                                                                         // #503
+    index=0;                                                                                             // #503
+    while (index < optable_optable_equivalence.length) // for each optable option                        // #503
+       {try      // separate entry into optable id part and associated optable value                     // #503
+           {entry=optable_optable_equivalence[index].toUpperCase();                                      // #503
+            i=entry.indexOf("=");                                                                        // #503
+            if (i == -1)                                                                                 // #503
+               {abort_error(40,"Missing equal sign in optable option definition " + entry);              // #503
+                }                                                                                        // #503
+            optable_option_id[index]=entry.substring(0,i);                                               // #503
+            optables_optable [index]=entry.substring(i+1);                                               // #503
+            // if the optable is an alias, issue error when the alias is not defined                     // #503
+            if (!optable_option_id[index].equals(optables_optable[index]))                               // #503
+               {index2=0;                                                                                // #503
+                while (index2 < index) // check all entries defined before this one                      // #503
+                   {if (optables_optable[index].equals(optables_optable[index2]))                        // #503
+                       {break;                                                                           // #503
+                        }                                                                                // #503
+                    index2++;                                                                            // #503
+                    }                                                                                    // #503
+                if (index2 >= index) // base optable is defined?                                         // #503
```

Comment:
"base optable is defined?" should be ??? "base optable is not defined?"

Reply:
I have changed the comment to: invalid index indicates not-found condition